source/getting_started/on-vm.rst

```diff
@@ -69,95 +30,39 @@ Reference the :ref:`build-os-image` section in the Getting Started Guide, and sp
 Create CaaS Virtual Disk
 ------------------------
 
-To create a virtual disk containing the CaaS partitions, the requirements are a USB flash drive and at least 16GB free disk space. Go through the following instructions to create and set up CaaS partitions on a *qcow2* formatted virtual disk.
+.. note::
+        This section is for flashing *flashfiles* package, no action required if you choose caas.img.
```

Comment:
s/no action required if you choose caas.img./no action **is** required if the image caas.img is used./

Reply:
will do.